packages/resolve-command/src/index.js

```diff
@@ -1,72 +1,83 @@
+import 'regenerator-runtime/runtime';
+
 function verifyCommand(command) {
-    if (!command.aggregateId) return Promise.reject('"aggregateId" argument is required');
-    if (!command.aggregateName) return Promise.reject('"aggregateName" argument is required');
-    if (!command.type) return Promise.reject('"type" argument is required');
+    if (!command.aggregateId)
+        return Promise.reject(new Error('"aggregateId" argument is required'));
+    if (!command.aggregateName)
+        return Promise.reject(new Error('"aggregateName" argument is required'));
+    if (!command.type) return Promise.reject(new Error('"type" argument is required'));
 
     return Promise.resolve(command);
 }
 
-function getAggregateState(aggregate, aggregateId, store) {
+function getAggregateState(aggregate, aggregateId, eventStore) {
```

Comment:
Due of previous issue, that function contains direct Promise. Operator await is a form for Promise invocation, but that function itself is not async